import.py

```diff
@@ -400,13 +400,19 @@ def import_c_file(
         sys.exit(1)
 
 
+def finalize_compile_command(compiler):
+    pieces = compiler.split("|")
+    pieces[0] += ' "$INPUT" '
+    return "|".join(pieces) + ' -o "$OUTPUT"\n'
+
+
 def write_compile_command(compiler: List[str], cwd: str, out_file: str) -> None:
     with open(out_file, "w", encoding="utf-8") as f:
         f.write("#!/usr/bin/env bash\n")
         f.write('INPUT="$(readlink -f "$1")"\n')
         f.write('OUTPUT="$(readlink -f "$3")"\n')
         f.write(f"cd {shlex.quote(cwd)}\n")
-        f.write(formatcmd(compiler) + ' "$INPUT" -o "$OUTPUT"\n')
+        f.write(finalize_compile_command(formatcmd(compiler)))
```

Comment:
I would prefer not to modify `formatcmd` since that modification is wrong in other situations (which use subprocess.run rather than outputting a shell script), and since post-processing its result may in theory do stuff to quoted `|`'s. How about this instead:
```py
def finalize_compile_command(cmdline):
    quoted = [arg if arg == "|" else shlex.quote(arg) for arg in cmdline]
    try:
        ind = quoted.index("|")
    except ValueError:
        ind = len(quoted)
    return " ".join(quoted[:ind] + ['"$INPUT"'] + quoted[ind:] + ["-o", '"$OUTPUT"'])
```

Reply:
Sounds good! Pushed the changes.